Dockerfile

```
@@ -0,0 +1,26 @@
+FROM apache/airflow:slim-latest-python3.10
+ARG ORCHESTRATION_PACKAGE
+COPY ${ORCHESTRATION_PACKAGE} ${ORCHESTRATION_PACKAGE}
+RUN pip install ${ORCHESTRATION_PACKAGE}
```

As recommended by hadolint:

```suggestion
RUN pip install --no-cache-dir ${ORCHESTRATION_PACKAGE}
```

It also complains that it would be good to have the package==version. I suspect this would help with the docker caching layers. Not critical for now

---

dropped the package, will install only dependencies to increase iteration speed (do not reinstall the package)